python/satyaml/ERMINAZ-1U.yml

```diff
@@ -12,3 +13,17 @@ transmitters:
     framing: QUBIK
     data:
     - *tlm
+  # New framing:
+  9k6 FSK CCSDS TMTC SPP downlink:
+    frequency: 435.775e+6
+    modulation: FSK
+    baudrate: 9600
+    deviation: 4800
+    framing: CCSDS Reed-Solomon
+    frame size: 128
+    RS basis: conventional
+    RS interleave depth: 1
+    scrambler: CCSDS
+    syncword threshold: 4
```

Comment:
I am not sure that this field has any effect. `gr_satellites_flowgraph.py` does not appear to read it. The syncword threshold is handled by a command line argument.